### PR TITLE
Post 1-10 doc updates

### PR DESCRIPTION
**Pull request by taoism4504**
- Added a link to the Azure-Secrets Engine doc in the Feature Deprecation table :mag: [Deploy Preview](https://vault-git-post-1-10-doc-changes-hashicorp.vercel.app/docs/deprecation)
- Added a known issue (add/modify Duo MFA method causes panic) to the release notes [Deploy Preview](https://vault-git-post-1-10-doc-changes-hashicorp.vercel.app/docs/release-notes/1.10#adding-modifying-duo-mfa-method-for-enterprise-mfa-triggers-a-panic-error)
- Added the three known issues to the Upgrade guide [Deploy Preview](https://vault-git-post-1-10-doc-changes-hashicorp.vercel.app/docs/upgrading/upgrade-to-1.10.x)

### CODE REVIEW

File: website/content/docs/upgrading/upgrade-to-1.10.x.mdx [hunk 1]
```diff
@@ -62,17 +62,35 @@ OIDC provider system to reduce configuration steps and enhance usability.
...
+- A "default" key for signing and verification of ID tokens
+- An "allow_all" assignment which authorizes all Vault entities to authenticate via a
+  client application
 
 If you created an [OIDC provider](/api-docs/secret/identity/oidc-provider#create-or-update-a-provider)
 with the name "default", [key](/api-docs/secret/identity/tokens#create-a-named-key) with the
 name "default", or [assignment](/api-docs/secret/identity/oidc-provider#create-or-update-an-assignment)
-with the name "allow_all" using the Vault 1.9 tech preview, the installation of these built-in
-resources will be skipped. We _strongly_ recommend that you delete any resources that have
+with the name "allow*all" using the Vault 1.9 tech preview, the installation of these built-in
```
Comment (mladlow) [inline website/content/docs/upgrading/upgrade-to-1.10.x.mdx:73]: This isn't formatting correctly but I'm not sure exactly what you need to change.

File: website/content/docs/upgrading/upgrade-to-1.10.x.mdx [hunk 2]
```diff
@@ -62,17 +62,35 @@ OIDC provider system to reduce configuration steps and enhance usability.
...
+resources will be skipped. We \_strongly* recommend that you delete any resources that have
 naming collisions before upgrading to Vault 1.10. Failing to delete resources with naming
 collisions could result unexpected default behavior. Additionally, we recommend reading the
 corresponding details in the OIDC provider [concepts](/docs/concepts/oidc-provider) document
 to understand how the built-in resources are used in the system.
+
```
Comment (mladlow) [inline website/content/docs/upgrading/upgrade-to-1.10.x.mdx:79]: Could you please nest these under a "Known Issues" header?

File: website/content/docs/upgrading/upgrade-to-1.10.x.mdx [hunk 3]
```diff
@@ -62,17 +62,35 @@ OIDC provider system to reduce configuration steps and enhance usability.
...
+## Single Vault follower restart causes election even with established quorum
+
+We now support Server Side Consistent Tokens (See [Replication](/docs/configuration/replication), [Vault Eventual Consistency](/docs/enterprise/consistency), and [Upgrade to 1.10](/docs/upgrading/upgrade-to-1.10.x).), which introduces a new token format that can only be used on nodes of 1.10 or higher version. This new format is enabled by default upon upgrading to the new version. Old format tokens can be read by Vault 1.10, but the new format Vault 1.10 tokens cannot be read by older Vault versions.
```
Comment (mladlow) [inline website/content/docs/upgrading/upgrade-to-1.10.x.mdx:82]: Ideally, these notes on this page and the release notes would be @include directives - you can find some examples on the 1.8.x upgrade page. That lets us make an update in only one place if we need to update the known issue announcement with fix version information.

**Review by mladlow — DISMISSED** [review]
I left three comments - the two short ones I think should be addressed but the @include directive one could be considered more of a thing to look out for in the future.

File: website/content/docs/upgrading/upgrade-to-1.10.x.mdx (hunk 1 above)
Comment (taoism4504) [reply website/content/docs/upgrading/upgrade-to-1.10.x.mdx:73]: I'm not sure either. I may have to do consult with other writers and see if there's a way to reformat it correctly.

File: website/content/docs/upgrading/upgrade-to-1.10.x.mdx (hunk 2 above)
Comment (taoism4504) [reply website/content/docs/upgrading/upgrade-to-1.10.x.mdx:79]: Done

File: website/content/docs/upgrading/upgrade-to-1.10.x.mdx (hunk 1 above)
Comment (mladlow) [reply website/content/docs/upgrading/upgrade-to-1.10.x.mdx:73]: I'm not 100% sure what you were going for but

```
the name "allow_all" using the Vault 1.9 tech preview, the installation of these built-in
resources will be skipped. We *strongly*
```

seemed to work:
Screen Shot 2022-03-23 at 17 48 54

Comment (mladlow) [reply website/content/docs/upgrading/upgrade-to-1.10.x.mdx:73]:
```
the name "allow\_all" using the Vault 1.9 tech preview, the installation of these built-in
resources will be skipped. We _strongly_
```

Also works if you escape the underscore in "allow_all" in the bullet list above, I think.

File: website/content/docs/upgrading/upgrade-to-1.10.x.mdx (hunk 3 above)
Comment (taoism4504) [reply website/content/docs/upgrading/upgrade-to-1.10.x.mdx:84]: OK Will fix this in a separate PR. Right now, I have to get this merged to fix some broken links.

**Review by aartiiyengar — APPROVED** [review]

**Comment by taoism4504** [comment]
Decided to open individual PRs to address each issue.

**Pull request closed without merging**